test/client/preparePayment.ts

```diff
@@ -11,17 +11,12 @@ const { preparePayment: REQUEST_FIXTURES } = requests;
 const { preparePayment: RESPONSE_FIXTURES } = responses;
 const RECIPIENT_ADDRESS = "rpZc4mVfWUif9CRoHRKKcmhu1nx2xktxBo";
 
-/**
- * Every test suite exports their tests in the default object.
- * - Check out the "TestSuite" type for documentation on the interface.
- * - Check out "test/client/index.ts" for more information about the test runner.
- */
-describe("preparePayment", () => {
+describe("client.preparePayment", function () {
   beforeEach(setupClient.setup);
   afterEach(setupClient.teardown);
 
   addressTests.forEach(function (test) {
-    describe(test.type, () => {
+    describe(test.type, function () {
```

Comment:
What's the reason behind this change or ESLint rule?

Reply:
I don't have a good enough understanding of JS/TS to really understand why, but for some reason, when I used arrow functions, the compiler complained about instance variables (like `this.client`). It went away when I switched to regular functions. I'm guessing it's something to do with scope.